compiler/rustc_const_eval/src/interpret/intrinsics.rs

```diff
@@ -4,6 +4,7 @@
 
 use std::assert_matches::assert_matches;
 
+use rustc_apfloat::Float;
```

Comment:
Please keep this qualified, it is only used a few times and it is useful to see immediately that this is an apfloat trait.